modules/swagger-codegen/src/test/java/io/swagger/codegen/javaPlayFramework/JavaPlayFrameworkOptionsTest.java

```diff
@@ -26,6 +47,55 @@ protected CodegenConfig getCodegenConfig() {
         return clientCodegen;
     }
 
+    private TemporaryFolder folder = new TemporaryFolder();
+
+    @BeforeMethod
+    public void setUp() throws Exception {
+        folder.create();
+    }
+
+    @AfterMethod
+    public void tearDown() throws Exception {
+        folder.delete();
+    }
+
+    @Test(description = "Test routes generated with more than one ids")
+    public void routeWithMoreIds() {
+        final File output = folder.getRoot();
+
+        Swagger swagger = new SwaggerParser().read("/home/gpastor/Code/baikal-adapter-br/timeline/conf/swagger.json");
```

Comment:
- This test only works on your laptop.
- If you think the test is relevant, you should try to contribute it back to the official swagger-codegen repository. This way others can benefit from it and our fork will be easier to maintain.

Reply:
You are just updating the fork with the current version of the original project so they probably have a test for it. I guess the test is not needed anymore

Reply:
Oh, my bad  I think I have changed the file route. Anyway, I think like @tzimisce012 if swagger codegen people are honorable devs they have to had a test for this feature.